src/plugins/workspace/public/components/workspace_form/utils.ts

```diff
@@ -335,11 +335,11 @@ export const validateWorkspaceForm = (
       }),
     };
   }
-  if (color && !validateWorkspaceColor(color)) {
+  if (color && (!validateWorkspaceColor(color) || !euiPaletteColorBlind().includes(color))) {
```

Comment:
nit: Since we have this `!euiPaletteColorBlind().includes(color)` check which inlcudes the logic for `!validateWorkspaceColor(color)`, we can remove `!validateWorkspaceColor(color)`.

Reply:
I‘ve tried it before but it crashed
```
 due to changes in "src/plugins/workspace/common/utils.ts"
server    log   [05:18:42.079] [fatal][root] ReferenceError: window is not defined
```